OffsetLib/azel.py

```diff
@@ -0,0 +1,32 @@
+import ktl
+import argparse
+
+from wftel import wftel
+import KeckLogger
+
+parser = argparse.ArgumentParser(description="Move the telescope in the az-el directions",
+                         usage="azel.py azoff eloff")
+
+parser.add_argument("azoff", type = float, help="az offset")
+
+parser.add_argument("eloff", type=float, help="el offset")
+
+
+args = parser.parse_args()
+
+
+def azel(x, y):
+    dcs = ktl.Service('dcs')
+    azoff = dcs['azoff']
+    eloff = dcs['eloff']
+    azoff.write(x, rel2curr = t)
+    eloff.write(y, rel2curr = t)
+    time.sleep(3)
```

Comment:
We'll want to make sure these sleeps are as short as possible since we don't want to waste time at night.  Why do we need a sleep right before `wftel`?